src/database/models/index.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const process = require('process');
+
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.js')[env];
+
+const basename = path.basename(__filename);
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
```

Comment:
```suggestion
if (config[env].url) {
  sequelize = new Sequelize(config[env].url);
} else {
  sequelize = new Sequelize(process.env.DEV_DATABASE_URL);
}
```

Reply:
Thanks for the suggestion, It's applied now.